client/src/app/products/product-form/product-form.component.html

```diff
@@ -65,8 +65,11 @@
 
         <mat-form-field>
           <mat-label>Product Store</mat-label>
-          <input matInput placeholder="Product Store" formControlName="store"
-                required data-test="productStoreInput">
+          <mat-select formControlName="store" required>
+            <mat-option value="Willies">Willies</mat-option>
+            <mat-option value="Pomme de Terre">Pomme de Terre</mat-option>
```

Comment:
This leaves out "RealFoodHub" (which is represented in the DB and the tests), and that needs to be fixed.

The order here is kinda random. I would go with something like alphabetical, but maybe with "Other" at the end?

I wonder if what we *really* want to do is have this list populated by a DB query that returns all the "known" stores? In general we want to avoid hard-coding lists like this, as that's really fragile.

Reply:
Ok. Real food hub only ocurres once in the data, so I thought it was an outlier.

Order is based on how common it is in the seed data (I assumed one store would be more common than the other, and that seed data would give an accurate representation of this)

I don't think it would be useful to make a database call to populate a static list.

Reply:
Added RFH, alphabetized

Reply:
Nice – thanks!